compiler/rustc_codegen_ssa/src/mir/operand.rs

```diff
@@ -45,9 +45,11 @@ pub enum OperandValue<V> {
     Immediate(V),
     /// A pair of immediate LLVM values. Used by wide pointers too.
     ///
-    /// An `OperandValue` *must* be this variant for any type for which
+    /// # Invariants
+    /// - for `Pair(a, b)`, `a` is always offset 0 but **may** be `FieldIdx::ONE`
```

Comment:
not entirely sure if this is more of a "by definition" thing or if it requires manually enforcing it. not entirely sure if anyone knows.

Reply:
Hm.

Is `b` guaranteed to *not* be at offset 0, then?

Reply:
better, worse, incorrect?